packages/shared/src/components/profile/ReadingTagProgress.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { ReactElement } from 'react';
+import { getDayOfYear, getDaysInYear } from 'date-fns';
+import { SimpleTooltip } from '../tooltips/SimpleTooltip';
+import { ReadingTopTag } from '../../graphql/users';
+
+interface ReadingTagProgressProps {
+  topTag: ReadingTopTag;
+  isFilterSameYear?: boolean;
+}
+
+const now = new Date();
+
+export function ReadingTagProgress({
+  topTag: { tag, readingDays, percentage },
+  isFilterSameYear = true,
+}: ReadingTagProgressProps): ReactElement {
+  const value = `${(percentage * 100).toFixed(0)}%`;
+  const denominator = isFilterSameYear ? getDayOfYear(now) : getDaysInYear(now);
```

Comment:
The denominator would be different if it is on the same year. In 2022, we're only 32 days of the year. So instead of `3/365 reading days` it would progress based on what day of the year we are currently at `3/32 reading days`. What do you think?

Reply:
Reading days is something you should receive from the server. It's not the number of days the user could have read articles. It's the number of days they actually read an article. We need to adjust the API to return this value as well. We can add another field `total` to the existing query.